src/exportable/ExportableConfiguration.ts

```diff
@@ -26,6 +27,16 @@ export class ExportableConfiguration implements Exportable {
     if (this.config.version) {
       yaml.contents.add({ key: 'version', value: this.config.version });
     }
+    if (this.config.dependencies) {
+      const fshDependencies: any[] = [];
+      this.config.dependencies.forEach((dependency: ImplementationGuideDependsOn) => {
+        fshDependencies.push({
+          [dependency.packageId]: dependency.version
```

Comment:
I'm not sure we want to push the `dependency.packageId` as an array. That seems to lead to output that looks like this:
```
dependencies:
  - hl7.fhir.us.core: 3.1.0
```
Which is not what SUSHI is expecting. SUSHI expected the output to look like this:
```
dependencies:
  hl7.fhir.us.core: 3.1.0
```
In this case `dependencies` is an object, not an array. So instead of making the `fshDependencies` array and pushing onto it, you could make `fshDependencies` an object, and add each `packageId` and `version` as a key and value pair on that object.

Reply:
Oh jeez

Reply:
@cmoesel just said this! So please ignore this thread.